spyder/app/mainwindow.py

```diff
@@ -4160,7 +4124,92 @@ def run_spyder(app, options, args):
 # Main
 #==============================================================================
 def main():
-    """Main function"""
+    """Main function."""
+    global CLI_OPTIONS
+    global CLI_ARGS

```

Comment:
These two variables can be declared as local because they are not used outside this function.

Reply:
Ok